### PR TITLE
Adds a new null rod form: the possessed dummy

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

This PR adds a new null rod form: the possessed dummy.

It's basically a possessed blade, but with the following differences:
It has a greatly reduced (in comparison to the possessed blade) force of 5 (and has worse combat stats in other areas as well).
It can be used as/like a ventriloquist dummy, but can't be renamed before or after being possessed by a ghost (in other words, you have to stick with the name that the ghost chose for the possessed dummy, and you can't rename the dummy without putting a ghost in it and having the ghost choose a name for you).
Upon being possessed by a ghost, the possessed dummy gains the TRAIT_NODROP trait- once it's on your hand and possessed, you can't take it off without doing something drastic like chopping off your arm.

## Why It's Good For The Game

Dummies are creepy, and Halloween is approaching. Also, ghosts possessing possessed dummies are probably going to scream for/demand bloodshed less than ghosts possessing possessed blades will, and can probably be used for some funny jokes/gimmicks.

Do note that the in-hand sprites for the dummy might be broken/weird currently; I don't know if icon_state and item_state override them.

## Changelog
:cl: ATHATH
add: Adds a new null rod form: the possessed dummy!
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
In order to not have the in-hand sprites get overridden, should I set the lefthand_file and righthand_file variables for the possessed dummy to "null"? I haven't really looked to sprite code all that deeply.

**Comment by PKPenguin321** [comment]
test it and find out

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -468,6 +468,62 @@
...
+	if(possessed)
+		return
+
+	to_chat(user, "<span class='notice'>You attempt to wake the spirit of the dummy...</span>")
+
+	possessed = TRUE
+
+	var/list/mob/dead/observer/candidates = pollGhostCandidates("Do you want to play as the spirit of [user.real_name]'s dummy?", ROLE_PAI, null, FALSE, 100, POLL_IGNORE_POSSESSED_BLADE)
+
+	if(LAZYLEN(candidates))
```
Comment (PKPenguin321) [inline code/game/objects/items/holy_weapons.dm]: im guessing this is all copypasted from the possessed blade which is bad

Comment (ATH1909) [reply code/game/objects/items/holy_weapons.dm]: most of it is, with some edits, yes

**Comment by Shadowflame909** [comment]
Hopefully, noob sec won't think I'm a ling when my possessed dummy keeps talking while I'm a corpse over here.

Good PR though!

**Comment by optimumtact** [comment]
we do not need more chaplain bloat, figure out an alternate way to get one in the game tbh

**Comment by ATH1909** [comment]
> we do not need more chaplain bloat, figure out an alternate way to get one in the game tbh

Maybe as lavaland loot?

**Comment by ATH1909** [comment]
> test it and find out

https://cdn.discordapp.com/attachments/326831214667235328/638141161390735360/test1.PNG

It looks like it has an in-hand sprite, but doesn't modify your sprite, if that makes any sense. I suspect that the normal ventriloquist's dummy has the same problem.

**Comment by fludd12** [comment]
From the title, I expected a Rubyquest reference.

**Comment by ATH1909** [comment]
yeah, I don't think that this is going anywhere

**Pull request closed without merging**